manifests/packages.pp

```diff
@@ -1,11 +1,11 @@
 class minecraft::packages {
 
-  if $minecraft::manage_java {
+  if $::minecraft::manage_java {
     class { '::java':
       distribution => 'jre',
       version      => 'latest',
     }
   }
 
-  ensure_resource('package', 'screen', {'ensure' => 'present' })
+  ensure_packages('screen')
```

Comment:
I'd be happy to wrap this in an `if $::minecraft::manage_screen` as well if required.